ocs_ci/ocs/node.py

```diff
@@ -908,6 +914,93 @@ def delete_and_create_osd_node_vsphere_upi(osd_node_name, use_existing_node=Fals
     return new_node_name
 
 
+def delete_and_create_osd_node_vsphere_lso(osd_node_name, use_existing_node=False):
```

Comment:
this function is specific to vsphere UPI. Better to have its name reflecting that

Reply:
Fixed